main.tf

```diff
@@ -5,11 +5,36 @@ provider "aws" {
 resource "aws_s3_bucket" "demo-s3" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure AWS S3 Buckets are accessible to all authenticated users**
 
**Vulnerability**
S3 bucket Access is allowed to all AWS Account Users.
 
**Remediation**
Make sure to note Amazon S3 bucket access control lists (ACLs) that provide read, write, or full-access to 'Everyone'. It is recommended to provide access to S3 buckets to only authenticated users instead.

source: accurics-devenv